docs/diagrams/ModelClassDiagram.puml

```diff
@@ -41,6 +41,9 @@ Person *--> Phone
 Person *--> Email
 Person *--> Address
 Person *--> "*" Tag
+Person *--> RiskAppetite
+Person *--> Birthday
+Person *--> Income
```

Comment:
Might need to add your new attributes into the Model package to keep them within the rectangle

Reply:
Wow I actually didn't know that's how it worked thanks for the tips!